app/models/miq_ae_method.rb

```diff
@@ -30,6 +30,22 @@ class MiqAeMethod < ApplicationRecord
   AVAILABLE_SCOPES     = ["class", "instance"]
   validates_inclusion_of  :scope,     :in => AVAILABLE_SCOPES
 
+  scope :name_path_search, lambda { |search|
+    where('name ILIKE ? or relative_path ILIKE ?', "%#{search}%", "%#{search}%") if search.present?
```

Comment:
```suggestion
    search.present? ? where('name ILIKE ? or relative_path ILIKE ?', "%#{search}%", "%#{search}%") : self
```

Alternatively, if we want no results, then you can do ` : none`

Do this across these 3 methods